Update advancedRP.md (#14083)
aspnet/AspNetCore.Docs: aspnetcore/includes/advancedRP.md

> For a more advanced introduction aimed at experienced developers or those familar with controllers and views, see [Introduction to Razor Pages](xref:razor-pages/index).

Reviewer: Typo familar --> familiar

You could consider omitting "experienced" since that's implied by "familiar with ...":

```suggestion
For a more advanced introduction aimed at developers who are familiar with controllers and views, see [Introduction to Razor Pages](xref:razor-pages/index).
```
